src/haversine.jl

```diff
@@ -24,10 +24,11 @@ function (dist::Haversine)(x, y)
     Δφ = φ₂ - φ₁  # latitudes
 
     # haversine formula
-    a = sind(Δφ/2)^2 + cosd(φ₁)*cosd(φ₂)*sind(Δλ/2)^2
+    # hypot(x, y) is equivalent to sqrt(x^2 + y^2)
+    sqrt_a = hypot(sind(Δφ/2), cosd(φ₁)*cosd(φ₂)*sind(Δλ/2))
```

Comment:
This is incorrect. The `cosd` coefficients are not squared, it's just the `sind` term.

Reply:
Ah whoops, I missed that somehow!  This is probably not very useful then :).